src/common/providers/identity.ts

```diff
@@ -640,9 +647,38 @@ function parseAdditionalUserInfo(decodedJWT: DecodedPayload): AdditionalUserInfo
     profile,
     username,
     isNewUser: decodedJWT.event_type === "beforeCreate" ? true : false,
+    recaptchaScore: decodedJWT.recaptcha_score,
   };
 }
 
+/** Helper to generate payload to GCIP from client request.
+ * @internal
+ */
+export function generateRequestPayload(
```

Comment:
*nit: I think the name "RequestPayload" also makes sense since the payload will be sent to GCIP, and honestly both make sense to me, but for the sake of consistency with existing naming conventions in the SDK let's call the payload a "ResponsePayload".

Reply:
+1 to calling it ResponsePayload.

Reply:
Updated